### DISCUSSION TITLE
Is there a way to show more stock information on stock allocation?

### DISCUSSION THREAD

**Discussion by An40n1** [inventree/InvenTree#4650] [General]
I usually sort and track my parts by purchase order but it doesn't show up when allocating stock on Build Orders, is there a way of displaying it? It shows the Batch, and I could just put the purchase order on Batch but I would prefer not to duplicate information and I would save a couple steps. 

![23423423](https://user-images.githubusercontent.com/84796251/233599808-e24d5b57-d5ff-4bd8-8dbf-f9b78a1bfdd6.png)

**Comment by SchrodingersGat** [comment]
@An40n1 it would be possible, but would require work to add it into the forms. All the data is there via the API. I'd be concerned that the allocation form would start to be come quite crowded?

**Reply by An40n1** [reply]
Maybe some toggles can be added at the Build Order settings to select what info you want to show up, and have them be turned off by default to avoid clutter.
